chips/msp432/src/sysctl.rs

```diff
@@ -0,0 +1,201 @@
+// System Controller (SYSCTL)
+
+use kernel::common::registers::{register_bitfields, ReadOnly, ReadWrite};
+use kernel::common::StaticRef;
+
+pub static mut SYSCTL: SysCtl = SysCtl::new();
+
+const SYSCTL_BASE: StaticRef<SysCtlRegisters> =
+    unsafe { StaticRef::new(0xE004_3000 as *const SysCtlRegisters) };
+
+#[allow(dead_code)]
+const WKEY: u8 = 0x69; // for writing to REBOOT_CTL register
+#[allow(dead_code)]
+const UNLKEY: u16 = 0x695A; // for unlocking IP protected secure zone
+#[allow(dead_code)]
+const UNLOCKED: u16 = 0xA596; // value if zone is unlocked
```

Comment:
Why include these if they aren't used?

Reply:
Of course I can remove them if you want, but these values are currently not used but are necessary for some register-accesses. I thought i define them although they are not used because then it's not necessary to look them up again in the datasheet.

Reply:
If you think they should be left in can you add a comment?

Someone will probably have to look at the datahseet to use them anyway so it still seems worth removing them if they are unused.

Reply:
Ok, then I'll remove them (: